app/scripts/controllers/meeting.details.ctrl.js

```diff
@@ -292,12 +297,22 @@ angular.module('dashboard')
             $rootScope.$emit(CONST.UNSAVEMEETINGDDATA, (self.hasUnsavedProposal || self.remarkIsUnsaved));
         });
 
-        $scope.$on('$destroy', unsavedRemarkWatcher);
-        $scope.$on('$destroy', unsavedProposalWatcher);
-        $scope.$on('$destroy', proposalCountWatcher);
+        var motionsWatcher = $rootScope.$on(CONST.MOTIONSUPDATED, function (aEvent, aData) {
+                $log.debug("meetingDetailsCtrl.motionsWatcher: ", arguments);
+                if (angular.isObject(aData)) {
```

Comment:
https://docs.angularjs.org/api/ng/type/$rootScope.Scope: "you can compare the newVal and oldVal. If these two values are identical (===) then the listener was called due to initialization."
Okay maybe the value difference check is better to be left out like you did?